### PR TITLE
Fix: Playerclasses for root flag #20

### PR DESCRIPTION
**Pull request by Rushaway**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -36,25 +36,25 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
-        if: github.ref == 'refs/heads/master' || github.ref == 'refs/heads/main'
+        if: github.ref == 'refs/heads/master' || github.ref == 'refs/heads/main' || github.ref == 'refs/heads/nide'
```
Comment (maxime1907) [inline .github/workflows/ci.yml:39]: remove nide ref

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -36,25 +36,25 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
-        if: github.ref == 'refs/heads/master' || github.ref == 'refs/heads/main'
+        if: github.ref == 'refs/heads/master' || github.ref == 'refs/heads/main' || github.ref == 'refs/heads/nide'
 
       - uses: dev-drprasad/delete-tag-and-release@v0.2.1
-        if: github.ref == 'refs/heads/master' || github.ref == 'refs/heads/main'
+        if: github.ref == 'refs/heads/master' || github.ref == 'refs/heads/main' || github.ref == 'refs/heads/nide'
```
Comment (maxime1907) [inline .github/workflows/ci.yml:42]: remove nide ref

File: .github/workflows/ci.yml [hunk 3]
```diff
@@ -36,25 +36,25 @@ jobs:
...
         with:
           delete_release: true
           tag_name: latest
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
 
       - uses: rickstaa/action-create-tag@v1
-        if: github.ref == 'refs/heads/master' || github.ref == 'refs/heads/main'
+        if: github.ref == 'refs/heads/master' || github.ref == 'refs/heads/main' || github.ref == 'refs/heads/nide'
```
Comment (maxime1907) [inline .github/workflows/ci.yml:50]: remove nide ref

File: .github/workflows/ci.yml [hunk 4]
```diff
@@ -36,25 +36,25 @@ jobs:
...
         with:
           tag: "latest"
           github_token: ${{ secrets.GITHUB_TOKEN }}
 
   release:
     name: Release
-    if: startsWith(github.ref, 'refs/tags/') || github.ref == 'refs/heads/master' || github.ref == 'refs/heads/main'
+    if: startsWith(github.ref, 'refs/tags/') || github.ref == 'refs/heads/master' || github.ref == 'refs/heads/main' || github.ref == 'refs/heads/nide'
```
Comment (maxime1907) [inline .github/workflows/ci.yml:57]: remove nide ref

File: src/addons/sourcemod/scripting/zr/zadmin.inc [hunk 5]
```diff
@@ -104,7 +104,7 @@ bool ZAdminMenu(int client)
     // Add items to menu.
     SetMenuTitle(menu_zadmin, title);
     AddMenuItem(menu_zadmin, "classmultipliers", classmultipliers, configdraw);
-    AddMenuItem(menu_zadmin, "weapons", weapons, configdraw);
+    AddMenuItem(menu_zadmin, "weapons", weapons, moderatordraw);
```
Comment (maxime1907) [inline src/addons/sourcemod/scripting/zr/zadmin.inc:107]: why this change

Comment (Rushaway) [reply src/addons/sourcemod/scripting/zr/zadmin.inc:107]: Bcs moderatordraw  (OperationType_Generic) can already restrict weapons by command line, allow them to do it via zadmin menu
https://github.com/srcdslab/sm-plugin-zombiereloaded/blob/298a89628aab9c49c9ff0fab2dc60d7b4d49f130/src/addons/sourcemod/scripting/zr/weapons/restrict.inc#L677

Comment (maxime1907) [reply src/addons/sourcemod/scripting/zr/zadmin.inc:107]: ok good point

**Comment by maxime1907** [comment]
solves https://github.com/srcdslab/sm-plugin-zombiereloaded/issues/20
